proposals/privacy.md

```diff
@@ -103,23 +102,28 @@ a particular person.  We will call these "privacy sensitive contexts".
 
 ### TDs MUST not contain PII.
 For example, a TD must not contain the email of a person with which it is associated or embed their name, address, or other PII 
-in a description or title field.
+in a description or title field, unless specifically required by the use cases. 
+ML: It must be under the descretion of the owner of the thing to chose to whom to expose which PII.
+There are cases where a TD without PII is useless.
 
 ### TDs MUST be transmitted only to Authenticated and Authorized consumers.
 Services providing TDs must require authentication (for example, using OAuth or PKI) of the requester to ensure they
 are who they say they are.  In addition, the system needs to ensure that the requester is authorized to receive TDs.
 
 ### Consumers MUST accept TDs only from Authenticated producers
-In other words, communication of TDs between producers and consumers must be over mutually authenticated channels.
+In other words, communication of TDs between producers and consumers must be over mutually authenticated channels,
+or the authenticity of a producer is validated by some other means (e.g. cryptographic signatures).
 
 ### TDs provided to a Consumer MUST be filtered to remove any optional information not required by that Consumer.
 For example, if a Consumer does not need human-readable titles or descriptions in a TD, these should be removed.
+ML: How does a producer know about the capabilities / requirements of a consumer? We don't have a mechanism to convey this information. In many scenarios a TD producer does not know about the potential consumers. This comment applies also o the points below.
 If a Consumer does not need to maintain state related to the consumed device, then an ID is not necessary and should be omitted.
 If a Consumer does not do semantic processing, then all semantic annotations can be omitted.
+ML: It still is useful for documentation purposes and should be preserved.
```

Comment:
One man's documentation is another man's privacy leak.   If data is not being used, then it should not be transmitted.  Of course a "master" TD in a database can retain it... for documentation.   Or a client can request it if it has a use for it.